minutes/2019-12-03.md

```diff
@@ -0,0 +1,79 @@
+# Jakarta EE Platform Call
+
+General
+Date: 2019-12-03
+
+Present:
+
+* Kevin Sutter (IBM)
+* Steve Millidge
+* Tanja Obradovic, Ivar Grimstad (Eclipse Foundation)
+* Ed Bratt, Bill Shannon, Dmitry Kornilov (Oracle)
+* David Blevins, Jonathan Gallimore (Tomitribe)
+* Carlos Andres De La Rosa(Eclipse Committer)
+* Emily Jiang (IBM)
+* Andy McCright (IBM)
+* Kenji Kazumura (Fujitsu)
+* Nathan Erwin (Eclipse Contributor)
+
+## Agenda
+
+### [KWS] Java SE 8 or Java SE 11 as the minimum.
+
+Another key decision to allow for continued development efforts.
+[KWS] Will need to propose a vote on the mailing list indicating the recommendation of Java SE 8 as the minimum, but we would support both Java SE 8 and 11.
+(Steve “volunteered” to post the vote.)
+
+**Discussion around Java SE Version**: 
+For inclusion in Jakarta EE 9 specification’s apis MUST support running on Java SE 8 and SHOULD support running on Java SE 11. 
+However compatible implementations of the Jakarta EE 9 Web Profile and Full Profile WON’T be required to support Java SE 8 and may certify compatibility on Java SE 11 and/or Java SE 8. 
+
+A portable application can use/depend on Java SE 8 APIs -> a compatible implementation must provide the SE 8 APIs when running on 11.
+
+Discussed introducing a Legacy profile that contains all the optional parts.
+
+| Web Profile (opt) | Full Profile (opt) | Legacy Profile (opt) |
+| ----------------- | ------------------ | -------------------- |
+| Java SE 11        | Java SE 11         | Java SE 8            |
+|                   |                    | Contains all Java SE 8 APIs dropped from 11 |
+
+The Legacy profile could possibly be shipped separately later.
+The question of who will implement the Legacy Profile was raised without being answered.
+
+Also discussed that an alternative to the legacy profile is to keep the “SE8 APIs” in the platform, but optional. 
+**“If you ship it, you test it!”** No legacy profile needed. 
+
+_The rest of the agenda was deferred to the next meeting._
+
+
+[SM] Decide which specifications to prune
```

Comment:
Some of these agenda items are probably not needed in future agendas

Reply:
Yes, I am copying them to next week's document